pipeline-runner/init.R

```diff
@@ -403,9 +408,10 @@ init <- function() {
         )
 
         tryCatchLog({
+                # Refresh pipeline_config with the new task input
+                pipeline_config <- load_config(input$server, input$apiVersion)
 
-
-                wrapper(input)
+                wrapper(input, pipeline_config)
```

Comment:
I moved the reload of config outside of the wrapper so that we can refresh the `pipeline_config` used in `init`.

If I didn't do this then when there was an error (which is not handled inside the `wrapper`) the `sns_topic`  would be chosen without knowing with api version we are working with, which means the api wouldn't receive the error message in the correct endpoint